packages/management/tkg-clusterclass-vsphere/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: tkg-clusterclass-vsphere.tanzu.vmware.com
+  namespace: tkg-clusterclass-vsphere
+spec:
+  displayName: "tkg-clusterclass-vsphere"
+  longDescription: "Clusterclass for Vsphere provider package defines the default clusterclass resource"
```

Comment:
Clusterclass -> ClusterClass
ClusterClass for vSphere defines the default set of vSphere ClusterClass resources for Tanzu